src/components/forms/PasswordResetForm/PasswordResetForm.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Button, Form } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import usersActions from '../../../_actions/users.actions';
+
+class PasswordResetForm extends React.Component {
+  state = {
+    data: {
```

Comment:
nit: In here you've used state = {data: { ... } } but in ForgotPassword you've just used state = { ... }. It would be nice if this was consistent